benchmark/benchmain/main.go

```diff
@@ -253,7 +258,9 @@ func init() {
 	flag.Var(&readMaxConcurrentCalls, "maxConcurrentCalls", "Number of concurrent RPCs during benchmarks")
 	flag.Var(&readReqSizeBytes, "reqSizeBytes", "Request size in bytes - may be a comma-separated list")
 	flag.Var(&readRespSizeBytes, "respSizeBytes", "Response size in bytes - may be a comma-separated list")
-	flag.StringVar(&memProfile, "memProfile", "", "Enables memory profiling output to the filename provided")
+	flag.StringVar(&memProfile, "memProfile", "", "Enables memory profiling output to the filename provided. \n"+
+		"To include every allocated block in the profile, set MemProfileRate to 1. To turn off profiling entirely, set MemProfileRate to 0."+
+		"524288 by default.")
```

Comment:
What is this about?

I'd think not setting --memProfile would turn off profiling?

How does setting memProfileRate to zero differ from its default of zero?

Reply:
Done. I was trying to do what runtime.MemProfileRate does, and pasting it's comment to here.

Now I change it: only when memProfile is set, runtime.MemProfileRate will read profile rate from it.